docs/config/dev/config.toml

```diff
@@ -0,0 +1,2 @@
+baseURL = "https://localhost:1313/"
```

Comment:
This (and the same in the staging file) allow for links to work properly across the app when not running the production site.